getting-started-tutorial/getting-started-tutorial.ipynb

```diff
@@ -2880,14 +2870,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Deploy the serving function"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Set the trained model and deploy the serving function"
+    "### Deploy the Serving Function\n",
+    "\n",
+    "Set the trained model and deploy the serving function."
```

Comment:
Does this work with the TOC extension?

Reply:
Yes. That's also what we do in the other NBs that I edited. This is a screen shot from the updated extension TOC::
![image](https://user-images.githubusercontent.com/27913885/102974904-f0aeb080-4507-11eb-800c-9efde4ada51a.png)
The line that you highlighted in your comment is in the same cell as the "Deploy the Serving Model Function" in Step 6, and as you can see, it appears in the TOC, as do all other headings that also have text in the same cell. The important thing is to start the cell with the heading and not with an anchor, as we previously did.

Reply:
okay, just verifying